zipkin-storage/elasticsearch/src/main/java/zipkin/storage/elasticsearch/NativeClient.java

```diff
@@ -149,9 +149,9 @@
   }
 
   final TransportClient client;
-  final boolean flushOnWrites;
+  private final boolean flushOnWrites;
```

Comment:
not required to change, but just so you know why I didn't mark these private. This is a package protected final internal class anyway, so we have no contract to uphold wrt changing thing. each private thing has potential to bloat class sizes and mixed visibility is distracting.

If you really want these things to be invisible to the package, a quicker and less bloaty way to do it is just to make the type private. At that point, you can't reference the fields and constructors anyway, so default visibility is fine. You'll notice this practice used in grpc I think..